Projects/Feature/History/Sources/Component/CheckBoxEditTitleBottomSheetView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CheckBoxEditTitleBottomSheetView.swift
+//  FeatureHistory
+//
+//  Created by 현수빈 on 11/26/24.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+import SharedDesignSystem
+
+struct CheckBoxEditTitleBottomSheetView: View {
+  @Bindable var store: StoreOf<HistoryDetailStore>
+  @FocusState var focus: Bool
+  var body: some View {
+    VStack(spacing: 13) {
+      TopNavigation(
+        centerTitle: "",
+        trailingItem: (Image.closeCross, {
+          store.send(.didTapDismiss)
+        })
+      )
+      HStack {
+        Text("내용 변경")
+          .notoSans(.headline)
+          .foregroundStyle(Color.greyScale950)
+        Spacer()
+      }
+      .padding(.horizontal, 20)
+      .padding(.vertical, 12)
+      
+      
+      InputField(
+        errorMessage: .constant(nil),
+        text: $store.newTitle,
+        placeHolder: store.selected?.label ?? "내용 변경",
+        isFocused: $focus
+      )
+      
+      
+      VStack {
```

Comment:
```suggestion
      VStack(spacing: 10) {
```

VStack의 default spacing은 항상 그렇지는 않지만, 일반적으로 약 8으로 알고 있어요! 피그마 상에선 10으로 되어있어서 수정 한번 부탁드립니다!